File: Node/models/block.js
class Block {
    constructor(index, 
        transactions, 
        difficulty, 
        prevBlockHash, 
        minedBy, 
        blockDataHash, 
        nonce, 
        dateCreated, 
        blockHash) {
            this.index = index;
            this.transactions = transactions;
            this.difficulty = difficulty;
            this.prevBlockHash = prevBlockHash.toString();
            this.minedBy = minedBy;
            this.blockDataHash = blockDataHash.toString();         
            this.nonce = nonce;
            this.dateCreated = dateCreated;
            this.blockHash = blockHash;
    }
}

module.exports = Block

File: BlockExplorer/BlockExplorer/Models/Block.cs
namespace BlockExplorer.Models
{
    using System;

    public class Block
    {
        public int Index { get; set; }

        public Transaction[] Transactions { get; set; }

        public int Difficulty { get; set; }

        public string PrevBlockHash { get; set; }

        public string MinedBy { get; set; }

        public string BlockDataHash { get; set; }

        public int Nonce { get; set; }

        public DateTime DateCreated { get; set; }

        public string BlockHash { get; set; }
    }
}
